### ISSUE TITLE
[Legitimate Site Unblocked] kraken.world

### ISSUE THREAD

**Issue by afateev1** [MetaMask/eth-phishing-detect#10015]
**Domain:*kraken.world*

**Details:** *Good afternoon, I recently purchased a domain, and when I am trying to visit my own website, you guys put it on the blacklist. What should I do? I will attach a screen of proof - of a domain purchase. We are a brand new website, and to be blocked right while haven't done anything yet)) Please help, thank you so much :)*

- [ YES] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)
@AlexHerman1 @409H thanks

**Comment by afateev1** [comment]
@AlexHerman1

**Comment by AlexHerman1** [comment]
nothing hosted here, i don't see any reason to unblock this site that was previously reported for phishing, when nothing is being hosted.

**Comment by AlexHerman1** [comment]
can you please explain the nature of this site in a new issue
